File: typescript/src/creational/builder/dto/Phone.ts
export enum TYPE_PHONE {
    MOBILE = "Mobile",
    HOME = "Home",
    WORK = "Work",
    OTHER = "Other"
}

export class Phone {
    private phone: string
    private typePhone: TYPE_PHONE

    constructor(phone: string, typePhone: TYPE_PHONE){
        this.phone = phone
        this.typePhone = typePhone
    }

    toString(): string{
        return `Phone: ${this.phone} - ${this.typePhone}`
    }
}

File: c#/Creational/Builder/Dto/Phone.cs
namespace Dto.Phone
{

    public enum TypePhone
    {
        Mobile,
        Home,
        Work,
        Other
    }
    public class PhoneDto
    {
        private string phoneNumber;
        private TypePhone typePhone;

        public PhoneDto(string phoneNumber, TypePhone typePhone)
        {
            this.phoneNumber = phoneNumber;
            this.typePhone = typePhone;
        }

        public override string ToString()
        {
            return $"{phoneNumber} - {typePhone}";
        }
    }
}